src/listener/utils/resource-transforms.js

```diff
@@ -23,6 +23,7 @@ THE SOFTWARE.
 import HipChatTransform from './transforms/hipchat';
```

Comment:
It will be nice when we can create one transform to cover all of the adapters where we want to use just plain-text responses.  We could avoid lots of duplicate code.

Reply:
Yeah, I totally agree. I think I'll create other issue about that.